tests/physics/runiface_parallel/runiface_parallel.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+#
+# TIn this test we compare results form runiface and runiface_parallel
+# using basic experiments. First we generate regular results, then parallel ones.
+#
+#
+# ###################################################################
+
+import numpy as np
+import sys
+import dreamtests
+
+from DREAM.DREAMSettings import DREAMSettings
+import DREAM.Settings.Equations.DistributionFunction as DistFunc
+import DREAM.Settings.Equations.IonSpecies as Ions
+import DREAM.Settings.Equations.RunawayElectrons as Runaways
+import DREAM.Settings.Solver as Solver
+import DREAM.Settings.CollisionHandler as Collisions
+from DREAM.DREAMException import DREAMException
+import DREAM
+
+def createExperimentData(index):
+    ds = DREAMSettings()
+    #ds.collisions.collfreq_type = Collisions.COLLFREQ_TYPE_COMPLETELY_SCREENED
+    ds.collisions.collfreq_type = Collisions.COLLFREQ_TYPE_PARTIALLY_SCREENED
+
+    # Physical parameters
+    E = 6       # Electric field strength (V/m)
+    n = 5e19    # Electron density (m^-3)
+    T = 100     # Temperature (eV)
+
+    # Grid parameters
+    pMax = 1    # maximum momentum in units of m_e*c
+    Np   = 300  # number of momentum grid points
+    Nxi  = 20   # number of pitch grid points
+    tMax = 1e-3 # simulation time in seconds
+    Nt   = 20   # number of time steps
```

Comment:
You could increase ``Nt`` to something like 200 to add some safety margin to the timeout test. At the moment, on my laptop (with compiler optimizations disabled), each simulation runs in about 0.7s while the timeout is 0.5s. On a faster computer, with compiler optimization, the simulation might finish before the timeout expires.

Actually, maybe it would be best to add ``Nt`` as an input parameter to ``createExperimentData()`` with default value 20. Then, in ``check_timeout()`` you can pass ``Nt=200`` instead just be sure that it will run for long enough.

Reply:
Good idea, I will add it in a minute.

Reply:
Please, check it now.

Reply:
The update looks good!